nailgun/entities.py

```diff
@@ -1042,6 +1042,7 @@ def __init__(self, server_config=None, **kwargs):
             'attach_stdin': entity_fields.BooleanField(),
             'attach_stdout': entity_fields.BooleanField(),
             'command': entity_fields.StringField(
+                default='top',
```

Comment:
Bonus. at least 95% of our tests don't care what command will be executed in container and are focused on different things, so i can't see the reason to manually specify exact command each time, considering cli factory is using default value for command too.